test/controllers/tasks_controller_test.rb

```diff
@@ -71,56 +70,109 @@
           completion_date: nil,
         },
       }
-
+      
       # Act-Assert
       expect {
         post tasks_path, params: task_hash
       }.must_change "Task.count", 1
-
+      
       new_task = Task.find_by(name: task_hash[:task][:name])
       expect(new_task.description).must_equal task_hash[:task][:description]
-      expect(new_task.due_date.to_time.to_i).must_equal task_hash[:task][:due_date].to_i
-      expect(new_task.completed).must_equal task_hash[:task][:completed]
-
+      expect(new_task.completion_date).must_equal task_hash[:task][:completion_date]
```

Comment:
`DEPRECATED: Use assert_nil if expecting nil from test/controllers/tasks_controller_test.rb:81. This will fail in Minitest 6.`